dask/array/routines.py

```diff
@@ -368,7 +399,7 @@ def apply_over_axes(func, a, axes):
     # Compute using `apply_along_axis`.
     result = a
     for i in axes:
-        result = apply_along_axis(func, i, result, 0)
+        result = apply_along_axis(func, i, result, None, None, 0)
```

Comment:
Should we extend this to `apply_over_axes`?

Reply:
I think so, but I wanted to defer changing `apply_over_axes` until we got `apply_along_axis` settled

Reply:
Sounds reasonable. We can circle back to this.